project/src/main/webapp/shared.js

```diff
@@ -55,31 +82,124 @@ async function moveTagsFromPoolToInput(clickedAdder, tagId) {
 
 /** Add tag to user input box once it is chosen */
 function addTagsToInput(clickedAdder, tagId) {
-  $.getScript('https://cdnjs.cloudflare.com/ajax/libs/bootstrap-tagsinput/0.8.0/bootstrap-tagsinput.js', function() {
-    $(tagId).on('itemAdded', function() {
-      if ($(tagId).prevAll().length > 2) {
-        const otherId = tagId === '#interests' ? '#skills' : '#interests';
-        $(otherId).prev().remove();
-        $(tagId).prev().prev().remove();
-      }
-    });
-    $(tagId).on('itemRemoved', function(event) {
-      addTagBackToPool(event.item, tagId);
-    });
+  $(tagId).on('itemAdded', function() {
+    if ($(tagId).prevAll().length > 2) {
+      const otherId = tagId === '#interests' ? '#skills' : '#interests';
+      $(otherId).prev().remove();
+      $(tagId).prev().prev().remove();
+    }
+  });
+  $(tagId).on('itemRemoved', function(event) {
+    addTagBackToPool(event.item, tagId);
+  });
+  getTagsScriptWithCallback(function() {
     $(tagId).tagsinput('refresh');
     $(tagId).tagsinput('add', $(clickedAdder).next().html());
   });
 }
 
+function getTagsScriptWithCallback(fn) {
+  $.getScript(
+      'https://cdnjs.cloudflare.com/ajax/libs/bootstrap-tagsinput/0.8.0/bootstrap-tagsinput.js',
+      fn);
+}
+
 /** Puts item back into pool if it was a preset */
 async function addTagBackToPool(item, tagId) {
   const prefilledItems = await getPrefilledInformation(tagId);
   if (prefilledItems.includes(item)) {
-    $(tagId + '-div > .prefilled-pool > .btn-group:contains("' + item + '")').show();
+    $(tagId + '-div > .prefilled-pool > .btn-group:contains("' + item + '")')
+        .show();
   }
 }
 
 /** Remove tag from pool of options once it is chosen */
 function removeTagsFromPool(clickedAdder) {
   $(clickedAdder).parent().hide();
 }
+
+function togglePrefilledSkills() {
+  $('#prefilled-skills').toggle();
+}
+
+function togglePrefilledInterests() {
+  $('#prefilled-interests').toggle();
+}
+
+/** Writes out relevant details to an event card */
+function populateEventContainer(event, containerId) {
+  const indexOfEventCard = 25;
```

Comment:
Is there a reason this is 25? I think the variable name is not clear - maybe rename to number of event cards.

Reply:
This is the location in the object that the event card shows up, and it's always at the 25th index